src/config/cozy-media-options.ts

```diff
@@ -1,4 +1,4 @@
-import { MediaOptions } from '@atlaskit/editor-core/src'
+import { MediaOptions } from '@atlaskit/editor-core'
```

Comment:
j'appellerai plutôt le commit quelque chose comme `refactor: Remove deeplink for @atlaskit/editor-core`